quadlet/minio-prometheus/minio.container

```diff
@@ -0,0 +1,26 @@
+[Unit]
+Description=A minio server contaienr for local S3 compatible storage
+After=local-fs.target network-online.target prometheus-network.service
+Requires=prometheus-network.service
+
+[Container]
+Image=docker.io/minio/minio:latest
+Exec=server --console-address ":9001"
+Environment=MINIO_ROOT_USER=admin
+Environment=MINIO_ROOT_PASSWORD=password
+Environment=MINIO_VOLUMES=/data
+Environment=MINIO_PROMETHEUS_URL='http://systemd-prometheus:9090'
+Environment=MINIO_PROMETHEUS_AUTH_TYPE='public'
+Environment=MINIO_PROMETHEUS_JOB_ID='minio-job'
+Network=bridge
```

Comment:
Why do you need the `bridge` network? (also in `prometheus.container`)

Reply:
I added this bridge network so the containers can have access to the internet.  I believe this is the default `podman` network, or "host" (although this may be wrong).

> I guess the Internal flag is part of the reason why you attach two networks. Can you please elaborate on what you're trying to achieve? I'm not sure there isn't a simpler way to achieve it.

The internal network was me trying to put all Prometheus monitoring in an isolated network.  It's likely this is not required, so can remove it if you prefer.  One (`bridge`) network provides internet access and the other (`prometheus`) network is used for monitoring.

Reply:
Have tried using the default network (removing `bridge` and `prometheus` from both files) and an error is thrown by minio - `Prometheus URL is unreachable`.  
I am in no way an expert in minio - in fact I first heard about it yesterday, so I'm happy to be told I'm doing something wrong.

Reply:
I tried removing both Networks and it seems to be working. Where did you get the error? Maybe I'm missing something. Having said that, separating the network is not a bad idea. Just don't make it `internal`.

Reply:
Sorry for the delay.  

I have changed the containers to use a single network in this commit - [e819931](https://github.com/containers/appstore/pull/13/commits/e81993119f58621d8d89a5422c01778854eace55).  The new network is not marked as internal and Prometheus works as expected now.

Reply:
Great